.rubocop.yml

```diff
@@ -79,6 +79,13 @@ Lint/UnexpectedBlockArity:
 Lint/UnmodifiedReduceAccumulator:
   Enabled: true
 
+Lint/UnusedMethodArgument:
+  Description: >-
+    Disabled as this can break YARD
+    See https://github.com/rapid7/metasploit-framework/pull/17735
+    Also see https://github.com/rubocop/rubocop/pull/11020
+  Enabled: false
+
```

Comment:
I think the rule still makes sense on `modules`

Reply:
Hmm, does RuboCop have a way to limit this to only library files or only files from a certain directory? This is a fair point, so I'll have to try find a solution for this 🤔

Reply:
@adfoster-r7 updated this to explicitly exclude the `lib/` directory. Can you let me know if you think this is sufficient?

Reply:
Looks good to me; cheers